src/utils/polyfill.ts

```diff
@@ -0,0 +1,97 @@
+import { shouldPolyfill } from '@formatjs/intl-numberformat/should-polyfill'
+
+export async function polyfill(locale: string) {
```

Comment:
Please use the same code style that we use throughout the site. (arrow function)

Reply:
Done